File: reto6/reto6.js
/** @param {string[]} gifts
 *  @returns {boolean} True if the gift is inside the box
 */
function inBox(box) {
  let isGiftInBox = false;
  for (let i = 1; i < box.length - 1; i++) {
    if (!box[i].includes("*")) continue;
    isGiftInBox =
      box[i].indexOf("*") > 0 && box[i].indexOf("*") < box[i].length - 1;
  }
  return isGiftInBox;
}

File: reto6/reto6.py
def in_box(box):
    isGiftInBox = False

    for i in range(1, len(box) - 1):
        if "*" not in box[i]:
            continue

        isGiftInBox = box[i].index("*") > 0 and box[i].index("*") < len(box[i]) - 1

    return isGiftInBox
